app/controllers/support/schools_controller.rb

```diff
@@ -6,8 +6,18 @@ def search
   end
 
   def results
-    @search_form = BulkUrnSearchForm.new(search_params)
-    @schools = policy_scope(@search_form.schools).includes(:preorder_information, :responsible_body)
+    if request.post?
+      @search_form = BulkUrnSearchForm.new(search_params)
+      @schools = policy_scope(@search_form.schools).includes(:preorder_information, :responsible_body)
+    elsif request.get?
+      @form = Support::NewUserSchoolForm.new(name_or_urn: params[:query])
```

Comment:
Not a blocker, but this feels a bit fragile - and also limits potential for using the endpoint from other places

Reply:
Agreed – the name of the form object isn't really reflective of its responsibilities so I've renamed it in f9e00d1

Reply:
:+1: